.github/actions/e2e_test/action.yml

```
@@ -258,6 +257,7 @@ runs:
         gcpProjectID: ${{ inputs.gcpProject }}
         gcpZone: ${{ inputs.regionZone || 'europe-west3-b' }}
         kubernetesVersion: ${{ inputs.kubernetesVersion }}
+        additionalTags: "run-name=${{ steps.create-prefix.outputs.prefix }}"
```

```suggestion
        additionalTags: "workflow=${{ github.workflow }}"
```
I think a better tag would be using the workflow run id
Resources are already named using `steps.create-prefix.outputs.prefix`.
If we tag them with their workflow ID, its easier to know where the resources came from.